Makefile

```diff
@@ -51,13 +51,24 @@ format: $(GO)
 	$(GO) fmt ./pkg/... ./cmd/...
 	$(GO) vet ./pkg/... ./cmd/...
 
-test: $(GO)
+test-build-host-local-plugin:
+	if [ ! -f $(BIN_DIR)/host-local -a `uname` = 'Linux' ]; then\
+		cd $(BIN_DIR);\
+		git clone https://github.com/containernetworking/plugins;\
+		cd plugins; git checkout v0.9.1; cd ..;\
+		./plugins/build_linux.sh;\
+		cp ./plugins/bin/host-local .;\
+		rm -rf plugins;\
+	fi
+	sudo rm -rf /var/lib/cni/networks/mynet
```

Comment:
We don't need this, do we?

Reply:
This is needed because ```/var/lib/cni/networks/mynet/last_reserved_ip.0``` has last reserved ip address.
It counts up like ```10.1.2.1```, ```10.1.2.2```, ...
And when it over the ```10.1.2.0/24``` range like 10.1.3.1, test fails because the test checks it.
https://github.com/k8snetworkplumbingwg/ovs-cni/blob/e3af30b64fce307c99d8328fbf26e0f9ba719b73/pkg/plugin/plugin_test.go#L332

Reply:
This must be a bug in our tests, no? When DEL is called, the CNI should release the IP. Have you tried running on top of https://github.com/k8snetworkplumbingwg/ovs-cni/pull/170?

Reply:
> Have you tried running on top of #170?

Yes. I am using rebased tree.

> This must be a bug in our tests, no? When DEL is called, the CNI should release the IP.

No. The PR #170 works. It releases the IP. If 
But ```host-local``` IPAM plugin does not reuse the IP.
You can confirm this by using ```cnitool```. https://github.com/containernetworking/cni/tree/master/cnitool

Reply:
Ouch, that sounds like a bug in host-local then ...

If this is the case, could we use `"dataDir"` as defined in https://www.cni.dev/plugins/current/ipam/host-local/, to keep the metadata under tmp and then safely remove it as part of the test suite?

Reply:
> could we use ```"dataDir"```

Ah, OK. I will imlement it.

Reply:
> And when it over the ```10.1.2.0/24``` range like ```10.1.3.1```, test fails because the test checks it.

Sorry, I have overlooked the ```ranges``` parameter to the host-local. Because of the parameter, after the ```10.1.2.254``` it go back to ```10.1.2.2```.
https://github.com/k8snetworkplumbingwg/ovs-cni/blob/e3af30b64fce307c99d8328fbf26e0f9ba719b73/pkg/plugin/plugin_test.go#L328

So we do not need ```sudo rm -rf /var/lib/cni/networks/mynet```. I will remove it.
But using ```"dataDir"``` is still good idea, so I will implement it at another PR.

Reply:
Fixed.